src/sentry/static/sentry/app/views/releases/detail/overview/chart/healthChart.tsx

```diff
@@ -162,6 +166,27 @@ class HealthChart extends React.Component<Props> {
     }
   }
 
+  getLegendTooltipDescription(serieName: string) {
+    const {platform} = this.props;
+
+    switch (serieName) {
+      case sessionTerm.crashed:
+        return getSessionTermDescription('crashed', platform);
+      case sessionTerm.abnormal:
+        return getSessionTermDescription('abnormal', platform);
+      case sessionTerm.errored:
+        return getSessionTermDescription('errored', platform);
+      case sessionTerm.healthy:
+        return getSessionTermDescription('healthy', platform);
+      case sessionTerm['crash-free-users']:
+        return getSessionTermDescription('crash-free-users', platform);
+      case sessionTerm['crash-free-sessions']:
```

Comment:
When I checked locally, I didn't ever get tooltips for this and 'crash free sessions'. The term key used here isn't the same as the keys being used in the term mapping. Instead of `crash-free-sessions` the mapping had `Crash Free Sessions` as the key.

Reply:
Oh I am still working on this PR, will push tests yet...but thanks for the heads up 😉